components/drops/OptimismHero.tsx

```diff
@@ -26,21 +25,25 @@ export const OptimismHero = () => {
 
   const chainId = useChainId();
 
-  const optimismGoerliQuery = useContractList(
-    OptimismGoerli.chainId,
-    getDashboardChainRpc(OptimismGoerli),
-    address,
+  const allContractListQuery = useAllContractList(address);
+
+  const optimismGoerliContracts = useMemo(
+    () =>
+      (allContractListQuery.data || []).filter(
+        (contract) => contract.chainId === OptimismGoerli.chainId,
+      ),
+    [allContractListQuery.data],
```

Comment:
this *smells* like a worse solution than before, assuming before we were only *querying* for opt goerli contracts here? Maybe I'm missing how it works but *if possible* we should only query for the contracts we need instead of querying for *all* and then filtering down after

Reply:
before we were only getting contracts from old registry which was wrong here but anyway we're not even using this page actively anymore, there's no way to get contracts from multichain registry _only_ for a chain, you get them all, so we need to filter if we only want a specific chain.